client/JSDebugger/configurationProvider.ts

```diff
@@ -119,8 +113,8 @@ export class MLConfigurationProvider implements vscode.DebugConfigurationProvide
         return config
     }
 
-    private async getAvailableRequests(username: string, password: string, servername: string, hostname: string): Promise<any> {
-        const url = `http://${hostname}:8002/jsdbg/v1/paused-requests/${servername}`
+    private async getAvailableRequests(username: string, password: string, debugServerName: string, hostname: string): Promise<string> {
+        const url = `http://${hostname}:8002/jsdbg/v1/paused-requests/${debugServerName}`
```

Comment:
1. Will the debug server support https?
2. Should we use a dedicated function instead of building each URL string on the fly?

Reply:
1. It should. I need to add the part.
2. Do you mean we have one function that builds url and every other function calls it with parameters?

Reply:
1. Ok.
2. Yes. But I'm not sure how much we can generalize that. Let's see how much more complex the code gets while fixing point 1.